databases/sqlite/package.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, makeWrapper
+, sqlite
+, sqlitePlugins
+, stdenvNoCC
+, symlinkJoin
+}:
+let
+  pathType = if stdenvNoCC.isDarwin then "DYLD" else "LD";
+
+in
+symlinkJoin {
+  name = "sqlite";
+  buildInputs = [ makeWrapper ];
+  passthru = {
+    libPaths = lib.mapAttrs (_: d: d.libPath) sqlitePlugins;
+  };
```

Comment:
This needs an explanation

Reply:
Also this shouldn't work, as there's the `override` and `overrideDerivation` attrs?

Reply:
```suggestion
  passthru = {
    libPaths = map (d: d.libPath) plugins;
  };
```